funcs_utils.php

```diff
@@ -337,16 +351,11 @@ function current_branch_cms_major(
     if (strpos($MODULE_DIR, '/developer-docs') !== false) {
         $currentBranch = cmd('git rev-parse --abbrev-ref HEAD', $MODULE_DIR);
         if (!preg_match('#^(pulls/)?([0-9]+)(\.[0-9]+)?(/|$)#', $currentBranch, $matches)) {
-            error("Could work out current major for developer-docs from branch $currentBranch");
+            error("Could not work out current major for developer-docs from branch $currentBranch");
```

Comment:
Not technically part of the purpose of this PR but it's a simple fix to make here and might save a minor headache later on